packages/components-js/tests/e2e/puppeteer/specs/link-tile-model-signature.e2e.ts

```diff
@@ -0,0 +1,235 @@
+import {
+  expectA11yToMatchSnapshot,
+  getActiveElementTagName,
+  getConsoleErrorMessages,
+  getConsoleErrorsAmount,
+  getLifecycleStatus,
+  getProperty,
+  initConsoleObserver,
+  selectNode,
+  setContentWithDesignSystem,
+  setProperty,
+  waitForStencilLifecycle,
+} from '../helpers';
+import type { Page } from 'puppeteer';
+
+let page: Page;
+beforeEach(async () => (page = await browser.newPage()));
+afterEach(async () => await page.close());
+
+const getHost = () => selectNode(page, 'p-link-tile-model-signature');
+const getRoot = () => selectNode(page, 'p-link-tile-model-signature >>> .root');
+const getOverlayAnchor = () => selectNode(page, 'p-link-tile-model-signature >>> a');
+const getPrimaryLink = () => selectNode(page, 'p-link-tile-model-signature > p-link[slot="primary"] ');
+const getSecondaryLink = () => selectNode(page, 'p-link-tile-model-signature > p-link[slot="secondary"] ');
+const getHeading = () => selectNode(page, 'p-link-tile-model-signature >>> .heading');
+const getDescription = () => selectNode(page, 'p-link-tile-model-signature >>> .description');
+
+const imgSrc =
+  'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyAQMAAAAk8RryAAAABlBMVEUAAAD2vP9xXLiUAAAAAXRSTlMAQObYZgAAABxJREFUGNNjYOBgYGBhYKAZ/R8MDsD4Q5amkz8ASp4PtTYYQZIAAAAASUVORK5CYII=';
+
+const initLinkTileModelSignature = (opts?: { description: string }): Promise<void> => {
+  const { description } = opts || {};
+
+  return setContentWithDesignSystem(
+    page,
+    `<p-link-tile-model-signature heading="Some heading"${description ? ` description="${description}"` : ''}>
+  <img src="${imgSrc}" alt="Some image label"/>
+  <p-link slot="primary" href="https://porsche.com/">Some label</p-link>
+  <p-link slot="secondary" href="#">Some label</p-link>
+</p-link-tile-model-signature>`
+  );
+};
+
+it('should mirror anchor props of slot name="primary" onto overlay anchor', async () => {
+  await initLinkTileModelSignature();
+
+  const primaryLink = await getPrimaryLink();
+  const overlayAnchor = await getOverlayAnchor();
+
+  expect(await getProperty(primaryLink, 'href')).toEqual(await getProperty(overlayAnchor, 'href'));
```

Comment:
Hmm, now with actual `p-link` children chances are high that people slot an `a` as the `p-link`'s child.  
Will this break the href mirroring?

Reply:
We talked about this before i think. We define which usage is supported, we want to replace the slotted `a` cases with a different approach and not support them in the future, so i would not support it here in a new component.